sdk-setup/src/sdk-manage

```diff
@@ -545,6 +545,13 @@ object_config_get()
     echo "${value:-$default}"
 }
 
+object_config_get_boolean()
+{
+    local value=
+    value=$(object_config_get "$@") || return
+    tr -d '0' <<<$value
+}
```

Comment:
This feels a bit like a hack. But it does work, and it is convenient. And with bash everything feels like a hack. So I guess it's good :)

Reply:
Yeah after all I did it a more traditional way :)

Reply:
LOL, I was thinking too fast. Reverted to the hacky way as that allows to separate error handling :)